### PR TITLE
feat: address management

### PR DESCRIPTION
**Pull request by mansipadgelwar**
Feature:
Address Management
- Add address
- Delete address

Minor fixes and error handling

### CODE REVIEW

File: src/backend/controllers/AddressController.js [hunk 1]
```diff
@@ -0,0 +1,141 @@
+import { Response } from "miragejs";
+import { formatDate, requiresAuth } from "../utils/authUtils";
+import { v4 as uuid } from "uuid";
+/**
+ * All the routes related to Address are present here.
```
Comment (MayankKumar10) [inline src/backend/controllers/AddressController.js:5]: remove the comment

File: src/components/CartCard/CartCard.js [hunk 2]
```diff
@@ -78,8 +78,10 @@ const CartCard = ({ product }) => {
           <button
             className="btn btn-primary-outline"
             onClick={() =>
+              Number(product.qty) > 3 &&
```
Comment (MayankKumar10) [inline src/components/CartCard/CartCard.js:81]: call these values in function not here

**Review by MayankKumar10 — APPROVED** [review]
Looks into these suggestions everything else looks good to me

**Review by Vishruta-Patil — APPROVED** [review]
Suggestion: use constants for action types instead of hard coding strings 
Clean Code, Looks good to merge

**Comment by khushijohri2001** [comment]
Remove the comments and try to apply the given suggestions also check the array if empty. Otherwise looks clean good to merge

**Pull request merged**